tool/install.sh

```diff
@@ -15,13 +15,9 @@ else
     echo "Node version: $(node --version)"
 fi
 
-travis_fold start install.npm_install
-  (set -x; npm install)
-travis_fold end install.npm_install
+(set -x; npm install)
 
 if [[ -z "$CI" || "$TASK" == *build* ]]; then
-  travis_fold start install.bundle
     bundle --version
     (set -x; bundle install)
-  travis_fold end install.bundle
```

Comment:
Might it be worth replacing the `travis_fold` calls by `echo` statements? This is what was done in https://github.com/dart-lang/site-www/pull/2921? For example:

> <img src="https://user-images.githubusercontent.com/4140793/114066245-81b45800-9869-11eb-8e39-adbd32c5fb3d.png" width=500>